packages/astro/src/@types/astro.ts

```diff
@@ -1532,6 +1532,37 @@ export interface AstroUserConfig {
 			 * - `"pathanme": The strategy is applied to the pathname of the URLs
 			 */
 			strategy: 'pathname';
+
+			/**
+			 * @docs
```

Comment:
```suggestion
```
I *think* simply deleting this will hide the entry! That's my intention, anyway, and this will be added back in when the feature is no longer experimental!